enclave_build/src/image_manager.rs

```diff
@@ -21,6 +22,8 @@ pub trait ImageManager {
     fn architecture(&self) -> Result<String>;
     /// Returns two temp files containing the CMD and ENV expressions extracted from the image
     fn extract_expressions(&self) -> Result<(NamedTempFile, NamedTempFile)>;
+    /// Returns true if the image manager uses the Docker daemon
+    fn use_docker(&self) -> bool;
```

Comment:
I find this method redundant, then the whole idea of making this trait abstract looses its sense.